app/src/main/res/layout/fragment_asset_info.xml

```diff
@@ -296,6 +296,8 @@
                 android:id="@+id/recycler_transactions"
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
+                android:paddingTop="@dimen/dimen_20dp"
```

Comment:
The same. Or stop name dimens like number or make it name like asset_text_side_dimen